anaconda_env/engine.yml

```diff
@@ -4,27 +4,32 @@ channels:
 - ooi
```

Comment:
Have you confirmed you can successfully run the following command to create a conda environment in which you can run the stream and ingest engines/handlers?
conda env create -f engine.yml

Reply:
I have built the engine conda environment via a "conda env create -f conda.yml" and verified that environment successfully runs stream_engine, ingest_engine, and the ingest-handlers (this was tested distinctly from the enginemeta bit below). In particular, I ran an ingest request (different from that used for the enginemeta test), and verified I could pull the newly ingested data out with both a sync and an async data request.